sdk/documentintelligence/azure-ai-documentintelligence/azure/ai/documentintelligence/_model_base.py

```diff
@@ -498,7 +512,7 @@ def __init__(self, *args: typing.Any, **kwargs: typing.Any) -> None:
     def copy(self) -> "Model":
         return Model(self.__dict__)
 
-    def __new__(cls, *args: typing.Any, **kwargs: typing.Any) -> "Model":  # pylint: disable=unused-argument
+    def __new__(cls, *args: typing.Any, **kwargs: typing.Any) -> Self:  # pylint: disable=unused-argument
```

Comment:
Is this expected?

Reply:
Yes, "Self" is equivalent to "Model" in this place.